src/styles/blocks/card.scss

```diff
@@ -0,0 +1,150 @@
+$button-color: #00acdc;
+$bg-color: white;
+$smalltext-color: #616070;
+$maintext-color: #060b35;
+
+* {
+  box-sizing: border-box;
+
+}
+
+h1,
+h2,
+p,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #d3d3d3;
+  background-color: $bg-color;
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+
+  &__image-wrapper {
+    width: 100%;
+    height: 100%;
+    margin-bottom: 0;
+    overflow: hidden;
+
+    .card__img {
+      width: 100%;
+      height: 100%;
+      object-fit: contain;
+      display: block;
+      background-image: url("../images/imac.jpeg") cover;
+    }
+  }
+
+  .card__title {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $maintext-color;
+  }
+
+  .product-code {
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $smalltext-color;
+    width: 100%;
+  }
+
+  .reviews {
+    display: flex;
+    align-items: center;
+    margin-top: 16px;
+
+    .stars {
+      display: flex;
+      justify-content: left;
+      align-items: left;
+
+      &__star {
+        width: 16px;
+        height: 16px;
+        background-size: cover;
+        background-image: url(../images/star.svg);
+      }
+
+      &__star:not(:first-of-type) {
+        margin: 0 2px;
+      }
+
+      &__star:first-of-type {
+        margin-left: 0;
+      }
+
+      &--4 {
+        .stars__star:nth-child(-n+4) {
+          background-image: url(../images/star-active.svg);
+        }
+      }
+    }
```

Comment:
Not fixed. Starts should works with modifiers 1-5
![2023-09-15_16h25_10](https://github.com/mate-academy/layout_catalog/assets/61904995/41a5f40c-e722-4195-9df4-036b79c3eb01)